Src/Test.UnitTests.Sarif.PatternMatcher/AnalyzeCommandTests.cs

```diff
@@ -294,6 +294,75 @@ public void AnalyzeCommand_PushInheritedDataTest()
             }
         }
 
+        [Fact]
+        public void AnalyzeCommand_WithDeprecatedName()
```

Comment:
>AnalyzeCommand_WithDeprecatedName [](http://example.com/codeflow?start=20&length=33)

You added a test! Nice. Note, that we could have tested this capability by simply extending another test, to make things more efficient.

There is always a balance, do you repeat a lot of work, just to check a small condition. Or do you try to inject that testing into some other test. It depends on how serious/significant the capability you are testing.

For this one, I probably would have just put the deprecated name in some other test and made sure that it flowed as appropriate.

Now a very detailed test specific to deprecated names probably would make sense for a test that was targeted at a lower level, the skimmer class, say, or tests against a specific method somewhere.